app/neo/Overview.tsx

```diff
@@ -21,6 +22,40 @@ type OverviewProps = {
   control?: ReactNode;
 };
 
+export const CreateNewArtefactButton = ({ title, open }: { title: string; open: () => void }) => {
+  const { addElement, importFromFile, importFromMarket } = useKnownHotkeys(title);
+  useHotkeys(addElement.hotkey, open, { keydown: false, keyup: true, scopes: ['neo'] });
+  const addTooltip = title.match('Import Projects') ? importFromFile.label + '\n' + importFromMarket.label : addElement.label;
+
+  return (
+    <div className='createNewButton'>
+      <Button
```

Comment:
not needed to wrap the button in a div, just add the class to the button if you want

```suggestion
    <Button className='createNewButton'
```